### PR TITLE
Spellchecking

### PR DESCRIPTION
**Pull request by pi1024e**
## Summary of the Pull Request
Since we are now doing spellchecking, I thought I'd help out.
 
## References
#4799 

## PR Checklist
* [X] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [X] Tests added/passed

## Detailed Description of the Pull Request / Additional comments
Spell Checks

### CODE REVIEW

**Comment by pi1024e** [comment]
Yes let us remove those

**Comment by pi1024e** [comment]
except some commands use the lowercase variant, and those commands are documented here, so... that is where things get tricky

**Comment by jsoref** [comment]
@pi1024e: You can add entries into `patterns` to specifically accept a long string (technically text matching pattern you added is removed before it parses things looking for unrecognized items).

An example pattern:
```
\b(?:)powershell -noprofile\b
```

---
I'm going to have to change how the output works, in theory you'd want to be told to remove entries from the whitelist, but right now you'll only see that if you add one misspelling or if you run w/ debug... For the time being, I'd suggest making an extra local branch in your repository on top of this branch and having it have a single misspelling -- you'd never create a PR from this branch nor merge it. You can then use the output to remove whitelist entries from this branch.

I was trying to avoid having the bot be too chatty, but I think people will probably be ok (or at least you would) w/ being told how to remove things from the whitelist.

**Comment by jsoref** [comment]
Right now the spell checker is actively ignoring `.yml` files. You could work on removing that exclusion:

https://github.com/microsoft/terminal/blob/31efd69149c3b23c000c7491f98254356b1eaf34/.github/actions/spell-check/excludes.txt#L52

I think the reason I excluded `.yml` files is that they often have shas in them, but a glance at this repository indicates that they don't, so one could (should?) remove that exclusion.

File: doc/specs/#754 - Cascading Default Settings.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ class IDynamicProfileGenerator
 ```
 
 For example, the WSL generator would return `Microsoft.Terminal.WSL`. The
-Powershell Core generator would return `Microsoft.Terminal.PowershellCore`.
+PowerShell Core generator would return `Microsoft.Terminal.PowershellCore`.
```
Comment (miniksa) [inline doc/specs/#754 - Cascading Default Settings.md]:
```suggestion
PowerShell Core generator would return `Microsoft.Terminal.PowerShellCore`.

```

File: src/cascadia/TerminalApp/GlobalAppSettings.cpp [hunk 2]
```diff
@@ -514,9 +514,9 @@ std::wstring_view GlobalAppSettings::_SerializeTabWidthMode(const TabViewWidthMo
 }
 
 // Method Description:
-// - Adds the given colorscheme to our map of schemes, using its name as the key.
+// - Adds the given ColorScheme to our map of schemes, using its name as the key.
```
Comment (miniksa) [inline src/cascadia/TerminalApp/GlobalAppSettings.cpp]: I don't believe this needs to be capitalized as the object name. I think the generic "color scheme" terminology was sufficient.

File: doc/specs/#754 - Cascading Default Settings.md (hunk 1 above)
Comment (jsoref) [reply doc/specs/#754 - Cascading Default Settings.md]: You probably don't want that extra blank line. GitHub changed the default behavior of suggestion such that on average suggestions include one -- fairly annoying.

**Pull request closed without merging**